### PR TITLE
Feature/style

### PR DESCRIPTION
**Pull request by Roamer15**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,1133 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  max-width: 1000pxpx;
```
Comment (nganifaith) [inline css/style.css]: remove the extra px. 
Most often leave the width to 100%

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,1133 @@
...
+  background-color: #fff;
+  margin: 0 18%;
```
Comment (nganifaith) [inline css/style.css]: margin  18% is too large,
You can change the width to 98% 
and leave the margin to auto

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,1133 @@
...
+  height: 100%;
+  font-size: small;
```
Comment (nganifaith) [inline css/style.css]: nope. Change font size by given specific values

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,1133 @@
...
+.main-line a {
+    color: #000;
+    text-decoration: none;
+}
+/*End of the heading news*/
+
+/*Start of main body*/
+.main-body {
+  display: grid;
+  grid-template-columns: 71% 29%;
```
Comment (nganifaith) [inline css/style.css]: use proper division values

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,1133 @@
...
+  box-sizing: border-box;
+  list-style: none;
+  display: flex;
+  padding-left: 5rem;
+}
+
+.option a {
+  text-align: center;
+  color: #7a7979;
+  text-decoration: none;
```
Comment (nganifaith) [inline css/style.css]: go through the nested lecture notes and implement it here

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,1133 @@
...
+}
+
+.date a {
+  text-decoration: none;
+  color: #000;
+}
+
+.logo {
+  width: 485px;
+  height: 64px;
```
Comment (nganifaith) [inline css/style.css:104]: same thing here, Nest the code

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,1133 @@
...
+.option-dropdown {
+  box-sizing: border-box;
+  list-style: none;
+  display: flex;
+  padding: 0 5rem 0 5rem;
+  margin-bottom: 15px;
+}
+
+.option-dropdown li {
+  padding-left: 1rem;
```
Comment (nganifaith) [inline css/style.css]: nest the css

**Review by nganifaith — DISMISSED** [review]

File: css/style.css (hunk 4 above)
Comment (Roamer15) [reply css/style.css]: i just noticed a problem though i divided it for just two division, i find a very wide space seperating them

Comment (Roamer15) [reply css/style.css]: image

**Pull request merged**